helpers/theme.js

```diff
@@ -33,12 +35,13 @@ const setTheme = (res) => {
 	const documents = defaultDocuments();
 	const baseDir = (res.locals.currentSchoolData || {}).documentBaseDir || documents.documentBaseDir;
 	const themeTitle = instanceSpecificTitle();
+	const shortThemeTitle = instanceSpecificTitle(true);
```

Comment:
where is the advantage of this whole implementation? the value for `Schul-Cloud Brandenburg` for example can be directly taken from the var `SC_TITLE`

Reply:
I wasn't sure if there ever is a difference between short_title and title so I figured if short_title is used in the headline, I might as well use short_title here too